containers/Loan/Issue/index.tsx

```diff
@@ -75,13 +87,11 @@ class IssueLoan extends React.Component<Props, State> {
 
   componentWillMount() {
     const { tokenId } = this.props;
```

Comment:
registry can also be set in case i was passed through the props from the issue page